### ISSUE TITLE
sluggable not working with symfony 2.0.6

### ISSUE THREAD

**Issue by diaspar** [doctrine-extensions/DoctrineExtensions#195]
I have this defined in my entity

   /**
 27      \* @var string $name
 28      *
 29      \* @Assert\NotBlank()
 30      \* @gedmo:Sluggable(slugField="slug")
 31      \* @ORM\Column(name="name", type="string", length=100, nullable=true)
 32      _/
 33     private $name;
 34 
 35     /_*
 36      \* @var string $slug
 37      *
 38      \* @gedmo:Slug
 39      \* @ORM\Column(name="slug", type="string", length=100, nullable=false, unique=true)
 40      */
 41      private $slug;

I made sure that doctrine extensions version 2.1.x is in vendors/

Basically the save occurs, but the slug field is not updated (nothing in it after the insert).

sluggable: true is defined in config.yml.

Timestable works fine in the same project.

Please confirm
